example/login_to_game.ts

```diff
@@ -1,28 +1,35 @@
-import { CertificateStore, convertNostaleToken, GfAccount, Identity } from "../src/index";
+import {
+    CertificateStore,
+    convertNostaleToken,
+    GameforgeAPI,
+    GfLang,
+    GfLocale,
+    Fingerprint,
+} from "../src/index";
 
 const email = "gf-login@gameforge.sucks.af";
 const password = "gfpls";
 const installationId = "8cd369b7-3f73-47c4-bf57-3544201ec275";
 
 async function main() {
-    const identity = new Identity("./identity.json");
+    const fingerprint = Fingerprint.fromFile("./identity.json");
 
-    // const certStore = CertificateStore.create("./cert.p12", "secret_gf_cert_password");
-    const certStore = CertificateStore.createFromPem("./all_certs.pem");
+    const certStore = CertificateStore.create("./cert.p12", "secret_gf_cert_password");
+    // const certStore = CertificateStore.createFromPem("./all_certs.pem");
 
-    const a = new GfAccount({
-        locale: "pl_PL",
-        gfLang: "pl",
+    const acc = new GameforgeAPI({
```

Comment:
After changing the class name to `GameforgeAPI` the name `acc` doesn't make any sense.

```suggestion
    const api = new GameforgeAPI({
```